ush/wave_grid_interp_sbs.sh

```diff
@@ -133,17 +133,18 @@ source "${USHgfs}/preamble.sh"
 
 # 1.b Run interpolation code
 
+  export pgm="${NET,,}_ww3_gint.x"
+  . prep_step
+
   set +x
-  echo "   Run ww3_gint
-  echo "   Executing ${EXECgfs}/ww3_gint
+  echo "   Executing ${pgm}"
   set_trace
 
-  export pgm=ww3_gint;. prep_step
-  ${EXECgfs}/ww3_gint 1> gint.${grdID}.out 2>&1
+  "${EXECgfs}/${pgm}" 1> gint.${grdID}.out 2>&1
   export err=$?;err_chk
 
 # Write interpolation file to main TEMP dir area if not there yet
-  if [ "wht_OK" = 'no' ]
+  if [ "wht_OK" = 'no' ]  # FIXME: This is never going to evaluate to true, wht_OK is a string and needs to be ${wht_OK}
```

Comment:
```suggestion
  if [ "${wht_OK}" = 'no' ]
```
Taking in suggestion